lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -139,18 +139,21 @@ class ReportRenderer {
    */
   _renderReport(report) {
     const container = this._dom.createElement('div', 'lh-container');
+    const isSoloCategory = report.reportCategories.length === 1;
 
     container.appendChild(this._renderReportHeader(report)); // sticky header goes at the top.
     container.appendChild(this._renderReportNav(report));
 
     const reportSection = container.appendChild(this._dom.createElement('div', 'lh-report'));
 
-    const scoreHeader = reportSection.appendChild(
-        this._dom.createElement('div', 'lh-scores-header'));
+    const scoreHeaderClass = `lh-scores-header ${isSoloCategory ? 'lh-scores-header__solo' : ''}`;
```

Comment:
well i was thinking that too, but our styling uses that header's margin to get out from under the pos:fixed header.